contracts/dtoken/tests/userflows/userflow.liquidation.feature

```diff
@@ -0,0 +1,47 @@
+Feature: User liquidation flow
+
+	Rule: Price for WETH, WNEAR are should be the same 5$
```

Comment:
@pahomas let's also mention volatility is 100% (btw, should volatility be 100% or 0% as volatility is Deviation 0 means no deviation)?

Reply:
The volatility was set as a ratio so the best value is 100%